src/main/java/org/valdroz/vscript/EquationParser.java

```diff
@@ -454,13 +455,18 @@ private BaseNode parseAsIdentifiable() {
             int mathFuncOpCode = determineBuildinFuncCode(word);
 
             if (isFunction) {
-                if (mathFuncOpCode > 0) {
+                if (mathFuncOpCode > 0 &&  mathFuncOpCode != 170){
                     node = newNode()
                             .withNodeOperation(mathFuncOpCode);
                     if (parseParamsFor(node, '(', ')', ',') == 0) {
                         node.addParameterNode(Constants.C_NULL);
                     }
-                } else {
+                } else if (mathFuncOpCode == 170) {
```

Comment:
`else if (mathFuncOpCode == 170)`  block is not reuquired

Reply:
removed `else if (mathFuncOpCode == 170)` block